modules/installation-adding-vcenter-root-certificates.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_vsphere/installing-vsphere-installer-provisioned.adocc
+
+[id="installation-adding-vcenter-root-certificates_{context}"]
+= Adding vCenter root CA certificates to your system trust
+
+Because the installation program requires access to your vCenter's API, you must add your vCenter's trusted root CA certificates to your system trust before you install an {product-title} cluster.
+
+.Procedure
+
+. From the vCenter home page, download the vCenter's root CA certificates. Click *Download trusted root CA certificates* in the vSphere Web Services SDK section. The `<vCenter>/certs/download.zip` file downloads.
+
+. Extract the compressed file that contains the vCenter root CA certificates. The contents of the compressed file resemble the following file structure:
++
+----
+certs
+├── lin
+│   ├── 108f4d17.0
+│   ├── 108f4d17.r1
+│   ├── 7e757f6a.0
+│   ├── 8e4f8471.0
+│   └── 8e4f8471.r0
+├── mac
+│   ├── 108f4d17.0
+│   ├── 108f4d17.r1
+│   ├── 7e757f6a.0
+│   ├── 8e4f8471.0
+│   └── 8e4f8471.r0
+└── win
+    ├── 108f4d17.0.crt
+    ├── 108f4d17.r1.crl
+    ├── 7e757f6a.0.crt
+    ├── 8e4f8471.0.crt
+    └── 8e4f8471.r0.crl
+
+3 directories, 15 files
+----
+
+. Add the files for your operating system to the system trust. For example, on a Fedora operating system, run the following command:
++
+----
+# cp certs/lin/* /etc/pki/ca-trust/source/anchors
+----
+
+. Update your system trust. For example, on a Fedora operating system, run the following command:
++
+----
+# update-ca-trust extract
```

Comment:
make sure that the command of "cp" and "update-ca-trust" should be performed as root privilege.

Reply:
The `#` symbol indicates commands to run as root in the collection. [From the docs guidelines](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#code-blocks)

> If the user must run a command as root, use a number sign, `#`, at the start of the command instead of a dollar sign, `$`.